addons/website/views/snippets/snippets.xml

```diff
@@ -462,8 +462,8 @@
     </div>
 
     <!-- Background -->
-    <t t-set="only_bg_color_selector" t-value="'section .row > div, .s_text_highlight'"/>
-    <t t-set="only_bg_color_exclude" t-value="'.s_col_no_bgcolor, .s_col_no_bgcolor.row > div, .s_masonry_block .row > div, .s_color_blocks_2 .row > div, .o_mega_menu .row > div, .s_image_gallery .row > div'"/>
+    <t t-set="only_bg_color_selector" t-value="'section .row > div, .s_text_highlight, .s_mega_menu_thumbnails_footer'"/>
+    <t t-set="only_bg_color_exclude" t-value="'.s_col_no_bgcolor, .s_col_no_bgcolor.row > div, .s_masonry_block .row > div, .s_color_blocks_2 .row > div, .s_image_gallery .row > div, .o_mega_menu .s_mega_menu_gray_area'"/>
```

Comment:
Is it actually useful to exclude gray areas? I understand the right side of the column would still be colored gray but I don't think it's really a problem. Besides, if the user re-order the column, then it should definitely not be excluded. So I think you can simply remove this.